modules/serverless-logic-install-web-console.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/install-serverless-operator.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-web-console_{context}"]
+= Installing the {ServerlessLogicOperatorName} from the web console
+
+You can install the {ServerlessLogicOperatorName} from the OperatorHub by using the {ocp-product-title} web console. Installing this Operator enables you to install and use serverless workflows.
```

Comment:
Not sure why, but this sounds better to me without "the". While "from Operator Hub" is more popular in the OCP docs, it seems that Serverless docs use "the". Your decision, of course. 

 ```suggestion
You can install the {ServerlessLogicOperatorName} from OperatorHub by using the {ocp-product-title} web console. Installing this Operator enables you to install and use serverless workflows.
```

Reply:
I will follow the OCP docs guidelines.